### ISSUE TITLE
🐛 [BUG] - user access to unauthorized link

### ISSUE THREAD

**Issue by Dun-sin** [dvstechlabs/Noteslify#228]
### Description

User still has access to the website without logging in, i crossed checked and this only happens when you directly paste this URL in the address bar → `https://noteslify.stonecss.com/myaccount`

### Reproduction steps

```bash
1. Go to private mode/window or a new browser you haven't logged in to before
2. Insert `https://noteslify.stonecss.com/myaccount` in the address bar
```
### Browsers

Chrome

### OS

Windows

**Comment by devarshishimpi** [comment]
Fixed in https://github.com/dvstechlabs/Noteslify/pull/229

**Comment by Dun-sin** [comment]
> Fixed in https://github.com/dvstechlabs/Noteslify/pull/229

Here's a trick, if you add closes #issue in PR template and someone makes a PR and fills in the issue, when it's merged it automatically closes the issue too

**Comment by devarshishimpi** [comment]
> > Fixed in #229
> 
> Here's a trick, if you add closes #issue in PR template and someone makes a PR and fills in the issue, when it's merged it automatically closes the issue too

Yes I am aware of that, but the PR wasn't made by me.
